rules/Line.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * This script conflates all linear features using Generic Conflation.
+ */
+
+"use strict";
```

Comment:
I'm just going to describe one of these new scripts and not add notes to every single script... `exports.tagThreshold` gives you the tag similarity threshold for tag matching. `exports.geometryType` allows for optimization with `ScriptMatchCreator`